src/theme.ts

```diff
@@ -253,12 +266,12 @@ export const newTheme = extendTheme({
         text: {
           primary: White, // Various MUI components and text
         },
-        Switch: {
-          defaultColor: TestTool, // switch ball off color
-        },
-        common: {
-          onBackground: TestTool, // switch track off color
-        },
+        // Switch: {
```

Comment:
These are overrides for MUI component colors.  I think it's a good idea to leave them in, because they are potentially helpful for future customization.  In this case, if we want to customize the colors of switches when they are turned off (currently a light gray by default)

I will add a comment explaining the purpose for them though

Reply:
OK, a comment explaining the purpose is a good solution.